Utilities/ReleaseScripts/scripts/singularity-check.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# Check if singularity is available and that unprivileged user namespace is enabled (needed for singularity-in-singularity)
+# It is needed for unit tests in modules HeterogeneousCore/SonicTriton and RecoEcal/EgammaClusterProducers
+# It avoids TritonService fallback server error
+
+if $(type singularity >& /dev/null); then
```

Comment:
just for my curiosity, is there a specific reason to put these command evaluations in `$()`?

Reply:
Mmm... I don't think so in this case since I am not assigning to any variable. Removing them also. Thanks!